cloud-controller-manager/do/loadbalancers.go

```diff
@@ -1453,6 +1458,26 @@ func getNetwork(service *v1.Service) (string, error) {
 	return network, nil
 }
 
+func getNetworkStack(service *v1.Service, lbType string, network string) (string, error) {
+	networkStack := service.Annotations[annDONetworkStack]
+	if networkStack == "" {
+		if network == godo.LoadBalancerNetworkTypeInternal || lbType == godo.LoadBalancerTypeRegionalNetwork {
```

Comment:
nit: I wonder if we should turn the default around, so that everything that isn't caught by a check, falls back to the original value. To avoid accidents.